exams/api.py

```diff
@@ -0,0 +1,28 @@
+"""
+API for exams app
+"""
+import hashlib
+
+from django.conf import settings
+
+
+def sso_digest(client_candidate_id, timestamp, session_timeout):
+    """
+    Compute the sso_digest value we need to send to pearson
+
+    Args:
+        client_candidate_id (int|str): id for the user, usually Profile.student_id
+        timestamp (int): unix timestamp
+        session_timeout (int): number of seconds the session will last
+
+    Returns:
+        str: the computed digest value
+    """
+    data = ''.join([
+        settings.EXAMS_SSO_PASSPHRASE,
```

Comment:
What this should do is raise `django.core.exceptions.ImproperlyConfigured` error if either of these are not set. The view should catch those and log with `log.exception` and return some sensible error response to the UI.